suite/tests/client-interface/cleancall-opt-shared.h

```diff
@@ -301,7 +301,7 @@ dump_diff_mcontexts(void)
     }
 
     dr_fprintf(STDERR, "Printing XMM regs:\n");
-    for (i = 0; i < NUM_SIMD_SLOTS; i++) {
+    for (i = 0; i < MCTX_NUM_SIMD_SLOTS; i++) {
```

Comment:
You used num-saved above but num-slots here: what is the reasoning?  In core/ you used num-saved for the various dump_* printing routines: why different here?

Reply:
Agree this is inconsistent. I think, since it's mcontext, it should be proc_num_simd_saved().